common.lic

```diff
@@ -197,9 +197,14 @@ module DRC
   module_function :get_noun
 
   def empty_hands
```

Comment:
Maybe just move this code there?

I'm trying to reduce the chance someone uses the wrong helper.